conda-store-server/conda_store_server/app.py

```diff
@@ -120,6 +120,12 @@ def _check_build_key_version(self, proposal):
         except Exception as e:
             raise TraitError(f"c.CondaStore.build_key_version: {e}")
 
+    win_extended_length_prefix = Bool(
+        False,
+        help="Use the extended-length prefix '\\\\?\\' (Windows-only), default: False",
```

Comment:
In general I prefer using `r''` strings whenever I have backslashes in the string, but this kind of thing is better enforced by linters and style formatters rather than code reviews. Only leaving it as a statement of preference for a potential discussion down the line, not asking for a change now.

```suggestion
        help=r"Use the extended-length prefix '\\?\' (Windows-only), default: False",
```